amid/nsclc.py

```diff
@@ -74,7 +75,7 @@ class NSCLC(Source):
 
     _root: str = None
 
-    # TODO: maybe move to filtering via `ignore_errors=True`?
+    # FIXME: move to filtering via `ignore_errors=True` + filtering via Filter (if needed)
     _INVALID_PATIENT_IDS = [
```

Comment:
do we still need this field? can't we just use --ignore-errors?

Reply:
I am not sure whether it causes error when populating or just produce unequal/empty masks afterwards. Further testing needed.